matrix_sdk/src/verification/requests.rs

```diff
@@ -15,7 +15,9 @@
 use matrix_sdk_base::crypto::{CancelInfo, VerificationRequest as BaseVerificationRequest};
 use ruma::events::key::verification::VerificationMethod;
 
-use super::{QrVerification, SasVerification};
+#[cfg(feature = "qrcode")]
+use super::QrVerification;
+use super::SasVerification;
 use crate::{Client, Result};
 
 /// An object controlling the interactive verification flow.
```

Comment:
The docs for the accept method for a VerificationRequest object lists the default methods that we advertise, we'll need to mention that this depends on the new feature flag and how this changes depending on it.

Furthermore the defaults are mentioned over here:
1. [UserIdentity](https://github.com/matrix-org/matrix-rust-sdk/blob/master/matrix_sdk/src/identities/users.rs#L82) in the sdk crate.
2. [Device](https://github.com/matrix-org/matrix-rust-sdk/blob/master/matrix_sdk/src/identities/devices.rs#L55) in the sdk crate.
3. [VerificationRequest](https://github.com/matrix-org/matrix-rust-sdk/blob/master/matrix_sdk_crypto/src/verification/requests.rs#L399) in the crypto crate.